.changeset/kind-poems-joke.md

```diff
@@ -0,0 +1,8 @@
+---
+"app-builder-lib": minor
+"builder-util-runtime": minor
+"electron-publish": minor
+"electron-updater": minor
+---
+
+add host option for keycloak provider
```

Comment:
What is `keycloak`? Maybe this could be changed to "Add `host` property for self-hosted Keygen instances

Reply:
Ups, `keycloak` is some other software I patched just before this pr 😅 So I mixed up the names ;-)